components/fieldlabel/package.json

```
@@ -17,22 +17,22 @@
   },
   "peerDependencies": {
     "@spectrum-css/icon": "^3.0.21",
-    "@spectrum-css/vars": "^8.0.0"
+    "@spectrum-css/tokens": "^1.0.1"
```

Let's bump this to the most recent release: 

```suggestion
    "@spectrum-css/tokens": "^1.0.4"
```

---

bumped up